public/index.html

```diff
@@ -26,8 +26,8 @@
     -->
     <link
       rel="stylesheet"
-      href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
```

Comment:
it won't let me comment where you haven't made changes but could you change the content string on line 10 to read "Flames of Exile" instead of "Web site created using create-react-app."  I know is insignificant but it is bugging me when pope shares the link.

Reply:
done